src/Gang/Gang.ts

```diff
@@ -325,7 +325,14 @@ export class Gang {
     if (this.members.length < numFreeMembers && this.respect < Math.pow(recruitCostBase, numFreeMembers)) {
       return numFreeMembers - this.members.length; // if the max possible is less than freeMembers
     }
-    return Math.floor(Math.log(this.respect) / Math.log(recruitCostBase)) + numFreeMembers - this.members.length; //else
+    return (
+      Math.min(
+        Math.floor(Math.log(this.respect) / Math.log(recruitCostBase)),
```

Comment:
A slight re-arrangement that both simplifies the expression and makes more intuitive sense (to me) is to bring `numFreeMembers` inside the `min` by adding it to both sides. This leaves an expression for total gang members on one branch, and `GangConstants.MaximumGangMembers` on the other.

Since that brings most of the stuff inside the min, I think you won't need the surrounding parens after.

Reply:
Since we're going at it with a butchers knife, I changed some of the existing declared variables to equal their constants via the constant's file instead of the odd hardcoded values of 3 and 5.

I accounted for the free members in the 'membersRecruitable' calc so we could remove the previous check.